python/mxnet/gluon/nn/basic_layers.py

```diff
@@ -464,3 +466,66 @@ def hybrid_forward(self, F, x):
 
     def __repr__(self):
         return self.__class__.__name__
+
+
+class Lambda(HybridBlock):
+    r"""Wraps an operator or an expression as a HybridBlock object.
+
+
+    Parameters
+    ----------
+    function : str or function
+        Function used in lambda must be one of the following:
+        1) the name of an operator that is available in both symbol and ndarray
+        2) an operator in either symbol or ndarray, and is shared by symbol and ndarray.
+        3) a function that conforms to "def function(data, *args, **kwargs)".
+
+
+    Positional Arguments
+    --------------------
+    Positional arguments to be used in function call.
+
+
+    Keyword Arguments
+    -----------------
+    Keyword arguments to be used in function call.
+
+
+    Inputs:
+        - **data**: input data. Its shape depends on the function.
+
+    Output:
+        - **out**: output data. Its shape depends on the function.
+    """
+    def __init__(self, function, *args, **kwargs):
```

Comment:
remove args/kwargs.
function=str should only support simple function without parameters